packages/block-editor/src/components/link-control/index.js

```diff
@@ -33,7 +33,35 @@ import { focus } from '@wordpress/dom';
 import LinkControlSettingsDrawer from './settings-drawer';
 import LinkControlSearchItem from './search-item';
 import LinkControlSearchInput from './search-input';
+import LinkControlSearchCreate from './search-create-button';
+const CREATE_TYPE = '__CREATE__';
```

Comment:
Would be good to have inline code comment to explain what it is.

Reply:
Agreed.

Reply:
Done.